packages/jest-types/src/Global.ts

```diff
@@ -51,12 +51,6 @@ export type EachTestFn<EachCallback extends TestCallback> = (
   ...args: Array<any>
 ) => ReturnType<EachCallback>;
 
-// TODO: Get rid of this at some point
-type Jasmine = {
-  _DEFAULT_TIMEOUT_INTERVAL?: number;
-  addMatchers: (matchers: Record<string, unknown>) => void;
```

Comment:
`addMatchers` implementation was removed in #9853

Reply:
I think `addMatchers` still exist on `jasmine`?

Reply:
Here I had in mind Jest globals. It was removed from Jest globals, but Jasmin types still have `addMatchers` here: https://github.com/facebook/jest/blob/0d0844a249a179197e82bd1ea097a4cb6dad9f32/packages/jest-jasmine2/src/types.ts#L91